crypto/crypto.go

```diff
@@ -166,13 +165,16 @@ func HexToECDSA(hexkey string) (*ecdsa.PrivateKey, error) {
 
 // LoadECDSA loads a secp256k1 private key from the given file.
 func LoadECDSA(file string) (*ecdsa.PrivateKey, error) {
-	buf := make([]byte, 64)
-	fd, err := os.Open(file)
+	stat, err := os.Stat(file)
 	if err != nil {
 		return nil, err
 	}
-	defer fd.Close()
-	if _, err := io.ReadFull(fd, buf); err != nil {
+	size := stat.Size()
+	if size != 64 {
```

Comment:
This is going to shoot you in the foot if someone uses an editor that adds a newline :P

Reply:
Now allowing an extra 1 or 2 characters of `\r` or `\n`